CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -91,17 +96,24 @@ class GroupsViewController: UIViewController {
     }
 
     private func setupConstraints() {
+        let backButtonSize = CGSize(width: 10, height: 18)
         let backSize = CGSize(width: 86, height: 20)
-        let nextBackPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 49)
-        let nextBottomPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 90)
+        let skipBottomPadding: CGFloat = Constants.Onboarding.skipBottomPadding
+        let nextBottomPadding: CGFloat = Constants.Onboarding.nextBottomPadding
         let nextButtonSize = CGSize(width: 225, height: 54)
         let searchSize = CGSize(width: 295, height: 42)
-        let searchTopPadding: CGFloat = 50
+        let searchTopPadding: CGFloat = 48
         let tableViewWidth: CGFloat = 295
         let tableViewBottomPadding: CGFloat = 57
```

Comment:
Not sure how others feel about this, but the wall of constants is hard to read imo. I'd either 1) hard code them in the constraints if only used once or 2) initialize the constants right before you use them instead of all at once. If you like it the way it is tho, that's fine